fe2-android/app/src/test/java/com/github/dedis/student20_pop/utility/security/HashTest.java

```diff
@@ -1,18 +1,11 @@
 package com.github.dedis.student20_pop.utility.security;
 
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThrows;
 
 import org.junit.Test;
 
 public class HashTest {
 
-  @Test
-  public void hashNullDataTest() {
```

Comment:
I don't have context, what was the issue with this test? superficially it seems fine

Reply:
I removed it because this was written with the old hashing function in mind but it does hold for the new one too.

Adding a commit that adds a tests for nulls and empty strings.